#!/usr/bin/python3

def raise_exception_msg(message=""):
	try:
		pass
	except Exception as e:
		raise e